tests/testthat/test-deployments.R

```diff
@@ -69,6 +69,14 @@ test_that("can read/write metadata", {
   expect_equal(out$meta2, "two")
 })
 
+test_that("can read/write version", {
```

Comment:
Where do we test reading a DCF file that lacks the `envVars` and `version` fields?

We should test reading single files with/without these fields as well as a collection of files where some of them lack these fields.